packages/sdk/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@pipcook/sdk",
+  "version": "1.0.0",
+  "description": "JavaScript SDK for accessing Pipcook Daemon",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "npm run clean && npm run compile",
+    "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo",
+    "compile": "tsc -b tsconfig.json"
+  },
+  "dependencies": {
+    "@pipcook/pipcook-core": "^1.0.1",
+    "axios": "^0.18.1",
+    "fs-extra": "^8.1.0",
+    "request": "^2.88.2"
```

Comment:
We need the `@types/*` packages for the above `axios`, `fs-extra` and `request` at `devDependencies`.

Reply:
`@type/axios` is not needed any more. look at:
https://www.npmjs.com/package/@types/axios